Sources/SwiftDocCUtilities/TemplateLibrary/Init/___FILEBASENAME_INIT___.docc/Documentation.md

```diff
@@ -0,0 +1,33 @@
+# Documentation
+
+@Metadata {
+  @TechnologyRoot
+}
+
+<!--@START_MENU_TOKEN@-->Summary<!--@END_MENU_TOKEN@-->
```

Comment:
I believe these tokens will only work in Xcode – should we consider not using them since a primary audience for this feature will be users of other text editors?

Reply:
Removed and changed for the following text:
```
# ___DOCUMENTATION_TITLE___ 

@Metadata {
  @TechnologyRoot
}

Add a single sentence or sentence fragment, which DocC uses as the page’s abstract or summary.

## Overview

Add one or more paragraphs that introduce your content overview. 
...
```